app/views/vacancies/search/_page_title_and_description.html.slim

```diff
@@ -2,4 +2,4 @@
   - content_for :page_title_prefix, landing_page.title
   - content_for :page_description, landing_page.meta_description
 - else
-  - content_for :page_title_prefix, t(".title", count: params[:page].to_i)
+  - content_for :page_title_prefix, t(".title", count: params[:page].to_i, heading: search_results_heading(vacancies_search: @vacancies_search, landing_page: @landing_page).gsub("&lsquo;", "").gsub("&rsquo;", ""))
```

Comment:
this logic is not to nice here `.gsub("&lsquo;", "").gsub("&rsquo;", ""`

Reply:
these &lsquo; characters might be disappearing anyway but that is another conversation and adam will be looking into this another time